modules/cli/pages/index.adoc

```diff
@@ -51,12 +49,257 @@ $ mv skupper $HOME/bin
+[id="linking-sites"] 
+== Linking sites
+
+A {service-network} consists of {skupper-name} sites.
+This section describes how to link sites to form a {service-network}.
+
+Linking two sites requires a single initial directional connection. However:
+
+* Communication between the two sites is bidirectional, only the initial linking is directional.
+* The choice of direction for linking is typically determined by accessibility. For example, if you are linking an OpenShift Dedicated cluster with a CodeReady Containers cluster, you must link from the CodeReady Containers cluster to the OpenShift Dedicated cluster because that route is accessible. 
+
+.Procedure
+
+. Determine the direction of the link. If both clusters are publicly addressable, then the direction is not significant. If one of the clusters is addressable from the other cluster, perform step 2 below on the addressable cluster.
+
+. Generate a token on the cluster that you want to link to:
++
+--
+----
+$ skupper token create <filename> 
+----
+
+where `<filename>` is the name of a YAML file that is saved on your local filesystem.
+
+This file contains a key and the location of the site that created it.
+
+NOTE: Access to this file provides access to the {service-network}. Protect it appropriately.
+--
+
+. Use a token to join the {service-network}:
+
+.. Create a link:
++
+----
+$ skupper link create <filename> [-name <link-name>]
+----
++
+where `<filename>` is the name of a YAML file generated from the `skupper token create` command and `<link-name>` is the name of the link.
+
+.. Check the link:
++
+----
+$ skupper link status
+Connection for conn1 not active
+----
++
+In this example no <link-name> was specified, the name defaulted to `conn1`.
+
+
+. If you want to delete a link:
++
+----
+$ skupper link delete <link-name>
+----
+where `<link-name>` is the name of the link specified during creation.
+
+
+// Type: assembly
+[id="exposing-services"] 
+== Exposing services on the {service-network}
+
+After creating a {service-network}, exposed services can communicate across that network.
+
+The `skupper` CLI has two options for exposing services:
+
+* `expose` supports simple use cases, for example, a deployment with a single service.
+See xref:exposing-simple-services[] for instructions.
+
+* `service create` and `service bind` is a more flexible method of exposing services, for example, if you have multiple services for a deployment.
+See xref:exposing-complex-services[] for instructions.
+
+
+// Type: procedure
+[id="exposing-simple-services"] 
+=== Exposing simple services on the {service-network}
+This section describes how services can be enabled for a {service-network} for simple use cases.
+
+.Procedure
+
+. Create a deployment, some pods, or a service in one of your sites, for example to create the backend service for the link:{getting-started-link}[tutorial]:
++
+----
+$ kubectl create deployment hello-world-backend --image quay.io/skupper/hello-world-backend
```

Comment:
You have the deployment created here, but at the `skupper expose` part of the procedure,
there is no example showing how to expose it.
If there won't be an example in this procedure, then I do not see the reason why showing
how to create the deployment here.

Reply:
yep, you're right, added example